app/views/layouts/application.html.slim

```diff
@@ -9,7 +9,8 @@ html
 
     = favicon_link_tag
 
-    meta name="apple-itunes-app" content="app-id=#{Rails.application.secrets.mobile_app_id["ios"]}"
+    /Removed due to the implementation of a custom smart banner
+    /meta name="apple-itunes-app" content="app-id=#{Rails.application.secrets.mobile_app_id["ios"]}"
```

Comment:
Just for reference: https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html